lib/srfi/144/math.stub

```diff
@@ -102,7 +102,8 @@
 
 (define-c double frexp (double (result int)))
 
-(define-c int (sign-bit "signbit") (double))
+(c-declare "#define sign_bit(v) (!!signbit(v))")
+(define-c int (sign-bit "sign_bit") (double))
```

Comment:
FYI you can just say:

`(define-c int sign-bit (double))`

Reply:
Oh, right. I've pushed an updated version that uses a shorter form.